git-hooks/pre-commit

```diff
@@ -7,12 +7,12 @@ const execa = require('execa');
 const chalk = require('chalk');
 
 execa
-    .stdout('git', ['diff', '--name-only', '--cached'])
+    .stdout('git', ['diff', '--name-only', '--cached', '--diff-filter=AM'])
     .then(staged => staged.split('\n'))
     // lint and commit any lint fixes
     .then(staged => {
         const jsFiles = staged
-            .filter(file => file.endsWith('.js'))
+            .filter(file => file.startsWith('static') && file.endsWith('.js'))
```

Comment:
Means it won't lint js files outside static? Maybe a regex that ignores scala.js would work better?